usr/share/rear/layout/save/GNU/Linux/510_current_disk_usage.sh

```diff
@@ -2,4 +2,4 @@
 # Public License. Refer to the included COPYING for full text of license.
 
 # Save the current disk usage (POSIX output format) in the rescue image
-df -Plh |grep -vE '^(encfs)' > $VAR_DIR/layout/config/df.txt
+df -Plh -x encfs -x tmpfs -x devtmpfs > $VAR_DIR/layout/config/df.txt
```

Comment:
@ProBackup-nl I understand your reasons behind it, but we need to be sure that it stays backwards compatible with e.g. SLES 10 and RHEL 4 to name a few very old Linux versions. Is something for next week.